index.js

```diff
@@ -35,18 +35,24 @@ async function main () {
     if (!await commandExists('openssl').then(() => true).catch(() => false)) throw new Error('You must have openssl in your path for this program to work.')
 
     if (!options.number && !options.image) throw new Error('Either a number or an image must be specified.')
-    
+
     if (options.image) options.number = (await transform(options.image, {
         pixels: options.pixels,
         width: +options.width,
         contrast: +options.contrast
     })).replace(/\n/g, '')
 
-    
+
     const result = await findPrime(options.number, options.sophie)
-    log.info(options.export === 'json' ? JSON.stringify(result, undefined, 2) : result.prime)
+
+    if (options.export === 'json')
+        log.info(JSON.stringify(result,undefined,2))
+    else if (options.export === 'prime')
+        log.info(result.prime)
+    else
+        result.prime.match(new RegExp(`.{1,${options.width * 2}}`,'g')).forEach((line) => console.log(line))
```

Comment:
Could we instead do a:
```
import { splitEvery } from 'ramda'  // at the top
// and in place of that, do: 
log.info(splitEvery(options.width * 2, result.prime).join('\n'))

Reply:
The log.info may print unnecessary symbols on the output (for coloring and other stuff) that's the reason for choosing console.log.